js/react.md

```diff
@@ -282,6 +282,51 @@ Save the file and run `amplify publish`. After the API is deployed along with th
 
 In your project directory, open `./amplify/backend/function` and you'll see the Lambda function that you created. The `app.js` file runs the Express function and all of the HTTP method routes are available for you to manipulate. For example, the `API.post()` in your React app corresponded to the `app.post(path, function(req, res){...})` code in this Lambda function. If you choose to customize the Lambda function, you can update it in the cloud using `amplify push`. 
 
+## Adding XR Sumerian Scene
+
+To load Amazon Sumerian scenes you will need to activate the [Amplify Auth category]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/js/authentication).
+
+```
+$ amplify add auth
+```
+
+The `./src/aws-exports.js` file that's created has all of the appropriate cloud resources defined for your application. Edit `./src/App.js` to include the Amplify library and configurations. Then, initialize the library as follows:
+
+```javascript
+import Amplify from 'aws-amplify';
+import aws_exports from './aws-exports';
+import { SumerianScene } from 'aws-amplify-react';
+import scene1Config from './sumerian_exports_<sceneId>'; // This file will be generated by the Sumerian AWS Console 
+
+Amplify.configure({...aws_exports,
+  XR: {
+    region: "us-east-1", // Sumerian region
+    scenes: { 
+      "scene1": { // Friendly scene name
+        sceneConfig: scene1Config // Scene configuration from Sumerian publish
+      }
+    }
+  }
+});
+```
+
+Update the `render()` method to include the Sumerian Scene component:
+
+Note: The UI component will inherit the height and width of the direct parent DOM element. Make sure to set the width and height styling on the parent DOM element to your desired size.
```

Comment:
Might want to make this an _include since it seems like the same info for everything